ui/package.json

```diff
@@ -29,12 +29,14 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "@types/circular-dependency-plugin": "^5.0.5",
     "@types/jest": "^27.5.2",
     "@types/node": "^17.0.45",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
     "@typescript-eslint/eslint-plugin": "^5.30.7",
     "@typescript-eslint/parser": "^5.30.7",
+    "circular-dependency-plugin": "^5.2.2",
```

Comment:
doesn't it make more sense to put it in the app package ?

Reply:
I had to think about it for a minute, but 👍 to this. Since `app` is the only package using `webpack` (and it has the devDependency on webpack right now), probably make sense to put it there for now. If we ever have multiple packages using webpack though, we'll want to move all those dev dependencies out here to the root.